x/crosschain/keeper/gas_payment_test.go

```diff
@@ -192,11 +192,15 @@ func setupZRC20Pool(
 	require.NoError(t, err)
 }
 
-func setAdminDeployFungibleCoin(ctx sdk.Context, zk testkeeper.ZetaKeepers, admin string) {
+func setAdminPolicies(ctx sdk.Context, zk testkeeper.ZetaKeepers, admin string) {
 	params := zk.ObserverKeeper.GetParams(ctx)
 	params.AdminPolicy = []*observertypes.Admin_Policy{
 		{
-			PolicyType: observertypes.Policy_Type_deploy_fungible_coin,
+			PolicyType: observertypes.Policy_Type_group1,
+			Address:    admin,
+		},
+		{
+			PolicyType: observertypes.Policy_Type_group2,
 			Address:    admin,
 		},
 	}
```

Comment:
It looks like one admin address argument is provided and it's used to setup both group1 and 2. Am I reading that right? Will changing it to use two different addresses in mainnet require a code change?

Reply:
This is a helper function for the gas payment unit tests. We need to deploy gas zrc20 to test the functionality, so this helper grant all right to an address, independently of the group, to simplify the tests